src/main/java/org/apache/accumulo/access/AccessEvaluator.java

```diff
@@ -159,13 +161,4 @@ static AccessEvaluator of(String... authorizations) {
     return new AccessEvaluatorImpl(AccessEvaluatorImpl.convert(authorizations));
   }
 
-  /**
-   * An interface that is used to check if an authorization seen in an access expression is
```

Comment:
This javadoc should be moved (with rewording) to the method that now takes a predicate.  Need to explain what the purpose of the predicate is, which this javadoc kinda does.

Reply:
Added to the javadoc in 4e80d25 to try to address this.